### DISCUSSION TITLE
fuzzy score for phrases

### DISCUSSION THREAD

**Discussion by opike** [nlp-compromise/nlp_compromise#899] [Q&A]
Apologies if I'm missing something obvious, but for this fuzzy match example:
https://observablehq.com/@spencermountain/compromise-fuzzy-matching?collection=@spencermountain/nlp-compromise#fuzzyMatch
Where it shows "Fuzzy score: .89", is that relying on API calls into the library? Or is that code (fuzzyMatch, editDistance, etc.) that has been pulled from inside of the library?

**Comment by spencermountain** [comment]
Hi oliver, yep, it makes no api calls and is computed in the library. The fuzzy part is a simple string comparison
Cheers

**Reply by opike** [reply]
Hi - Thanks for the response. 
Is there a method signature where 2 strings are passed in and the fuzzy score is returned?

**Reply by spencermountain** [reply]
i would recommend using something like this - https://github.com/tad-lispy/node-damerau-levenshtein

**Comment by opike** [comment]
I'll check it out - thanks!
